js/layout-javascript/simple-list-code.js

```diff
@@ -949,6 +949,63 @@ DynamicList.prototype.prepareData = function(records) {
   return records;
 }
 
+DynamicList.prototype.convertFilesForComments = function(users) {
```

Comment:
Maybe our next task about optimizing LFD will be more of a development optimization one and it could be to split all the common/repeated functions out in a separate module and import it through webpack @hcarneiro  @squallstar @tonytlwu

Reply:
I could probably improve that specific function you highlighted to use a similar that already exists. Just probably need a new parameter to make the distinction. Let me see what I can do.